Student/Student/Courses/Syllabus/SyllabusViewController.swift

```diff
@@ -30,6 +32,13 @@ class SyllabusViewController: UIViewController {
     var syllabus: CoreWebView?
     var assignments: SyllabusActionableItemsViewController?
 
+    lazy var relayoutAssignmentConstraints: Void = {
```

Comment:
This is a `lazy var` so that it only gets triggered once, is that right?

Reply:
correct.